tests/rest/media/v1/test_url_preview.py

```diff
@@ -562,3 +565,126 @@ def test_accept_language_config_option(self):
             ),
             server.data,
         )
+
+    def test_oembed_url(self):
+        """Test changing a URL via oEmbed."""
+        # Route the HTTP version to an HTTP endpoint so that the tests work.
+        with patch.dict(
+            "synapse.rest.media.v1.preview_url_resource._oembed_patterns",
+            {
+                re.compile(
+                    "http://twitter\\.com/.+/status/.+"
+                ): "http://publish.twitter.com/oembed",
+            },
+            clear=True,
+        ):
+
+            self.lookups["publish.twitter.com"] = [(IPv4Address, "8.8.8.8")]
+            self.lookups["cdn.twitter.com"] = [(IPv4Address, "8.8.8.8")]
```

Comment:
Probably best to use one of the reserved example IP addresses in tests, in case something goes wrong and leaks out to the internet.

Reply:
This is copied from the tests above, so it is being consistent.

Reply:
Looks like this is the only test that does this weirdly, I'll update it.